src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public static final int MAX_RADIUS;
+
+    static {
+        MAX_RADIUS = 100;
+    }
```

Comment:
why do you need this variable here. It is better to create it in the class, then you will use it. likewise in the following cases